data_updates/R/weo.R

```diff
@@ -25,11 +25,12 @@ table.name = "weo"
 table.quote = c("repo",table.name)
 
 # Load data, removing na strings
-# data_url = "https://www.imf.org/external/pubs/ft/weo/2018/01/weodata/WEOApr2018all.xls"
-data_url = "https://www.imf.org/external/pubs/ft/weo/2019/01/weodata/WEOApr2019all.xls"
-weo = read.csv(data_url,sep="\t",na.strings=c("","n/a","--"))
+data_url = "https://www.imf.org/~/media/Files/Publications/WEO/WEO-Database/2020/02/WEOOct2020all.ashx"
+weo = read.csv(data_url,sep="\t",na.strings=c("","n/a","--"),fileEncoding = "utf-16")
+weo$X = NULL
+weo = subset(weo,WEO.Country.Code!="International Monetary Fund, World Economic Outlook Database, October 2020")
```

Comment:
Shall we keep editing these lines whenever the URL or encoding changes? I am thinking of passing these as arguments to the R script, or as part of a config file.